docs/modules/tiered-storage/pages/overview.adoc

```diff
@@ -7,22 +7,25 @@
 
 {description}
 
-NOTE: If your Enterprise license was generated before Hazelcast Platform version 5.2, you'll need a new Enterprise license that enables Tiered Storage. See xref:deploy:enterprise-licenses.adoc[].
+NOTE: If your Enterprise license was generated before Hazelcast Platform version 5.2, you'll need a new Enterprise license that enables Tiered Storage.
+See xref:deploy:enterprise-licenses.adoc[].
 
-== What Can You Do with Tiered Storage?
+Tiered Storage allows you to work with datasets that are larger than memory using the same Hazelcast map API.
+It allows you to optimize infrastructure costs and performance by combining a fast, high performance in-memory layer and a lower-cost, high-capacity disk-based storage layer.
 
-Have fewer Hazelcast members for large maps. For example, for a map with 1 TB of data, you'll need
-ten Hazelcast members if all data is kept in memory and each member has 100 GB of data. Tiered Storage allows you to split the data between memory and disks, which requires fewer cluster members depending on how the data is distributed.
+Tiered Storage allows you to work with datasets that are larger than memory using the same Hazelcast map API. It allows you to optimize infrastructure costs and performance by combining a fast in-memory layer with a cost-effective, high-capacity disk-based storage layer.
```

Comment:
>Tiered Storage allows you to work with datasets that are larger than memory using the same Hazelcast map API. It allows you to optimize infrastructure costs and performance by combining a fast in-memory layer with a cost-effective, high-capacity disk-based storage layer.

This part seems duplicate information. Previous paragraph also has same info.

Reply:
Removed the duplicate.